index.js

```diff
@@ -169,5 +166,49 @@ function handleArguments(env) {
   }
 
   // Load and execute the CLI version
-  require(path.join(__dirname, '/lib/versioned/', range, '/'))(opts, env);
+  require(path.join(__dirname, '/lib/versioned/', range, '/'))(opts, env, disp);
+}
+
+function customize(opts, env, disp) {
+  var specifiedOpts = customization.getSpecifiedOpts(opts, specifiedFlags);
+  var specifiedEnv = customization.getSpecifiedEnv(env, specifiedFlags);
+
+  // Map an array of keys to preserve order
+  var configFilePaths = ['home', 'cwd'].map(function(key) {
+    return env.configFiles['.gulp'][key];
+  });
+
+  configFilePaths.filter(isString).forEach(function(filePath) {
+    var config = require(filePath);
+    customizeByEachConfig(config, opts, env, disp, filePath);
+  });
+
+  merge(opts, specifiedOpts);
+  merge(env, specifiedEnv);
+}
+
+function customizeByEachConfig(config, opts, env, disp, filePath) {
+  try {
+    var config = require(filePath);
```

Comment:
I  found that this line is redundant with line 182. I'll remove this line or line 182.